packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -73,9 +73,9 @@ type TRawProject = {
   owner: {
     id: string;
   };
-  permissions: TApplicationContextPermissions;
-  actionRights: TApplicationContextActionRights;
-  dataFences: TApplicationContextDataFences;
+  permissions: TApplicationContextPermissions | null;
+  actionRights: TApplicationContextActionRights | null;
```

Comment:
Hm. Why are they nulled and not `{}`, now? Any specific reason?

Reply:
https://github.com/commercetools/merchant-center-application-kit/pull/973#discussion_r313384958